File: Python/String-2/double_char.py
# Given a string, return a string where for every 
# char in the original, there are two chars.


# double_char('The') → 'TThhee'
# double_char('AAbb') → 'AAAAbbbb'
# double_char('Hi-There') → 'HHii--TThheerree'
def double_char(str):
  res = ""
  for i in str:
    res=res+i
    res=res+i
    
  return res
    

File: java/String-2/doubleChar.java
//Given a string, return a string where for every char in the original, there are two chars.

// doubleChar("The") → "TThhee"
// doubleChar("AAbb") → "AAAAbbbb"
// doubleChar("Hi-There") → "HHii--TThheerree"

public String doubleChar(String str) {
  String res = "";
  for(int i = 0; i < str.length(); i++) {
    res += str.substring(i,i+1);
    res += str.substring(i,i+1);
  }
  return res;
}
